dtfs-central-api/api-tests/mocks/entities/utilisation-report-entity.ts

```diff
@@ -11,4 +11,26 @@ export const anUploadedUtilisationReportEntity = (): UtilisationReportEntity =>
   mock.data = [];
   mock.updatedByUserId = 'SYSTEM';
   return mock;
+};
+
+export const aNotReceivedUtilisationReportEntity = (): UtilisationReportEntity => {
+  const mock = new UtilisationReportEntity();
+  mock.bankId = '123';
+  mock.azureFileInfo = undefined;
+  mock.reportPeriod = aReportPeriodPartialEntity();
+  mock.status = UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED;
+  mock.data = [];
+  mock.updatedByUserId = 'SYSTEM';
+  return mock;
+};
+
+export const aNonUploadedMarkedReconciledUtilisationReportEntity = (): UtilisationReportEntity => {
+  const mock = new UtilisationReportEntity();
+  mock.bankId = '123';
+  mock.azureFileInfo = undefined;
+  mock.reportPeriod = aReportPeriodPartialEntity();
+  mock.status = UTILISATION_REPORT_RECONCILIATION_STATUS.RECONCILIATION_COMPLETED;
+  mock.data = [];
+  mock.updatedByUserId = 'SYSTEM';
```

Comment:
A report which has been marked as completed would have `updatedByUserId` be equal to `'TFM-<userId>'`